src/librustc_mir/build/expr/mod.rs

```diff
@@ -75,5 +75,5 @@ mod as_lvalue;
 mod as_rvalue;
 mod as_operand;
 mod as_temp;
-mod category;
+pub mod category;
```

Comment:
Probably not. I think I added it because I was using it in `block.rs`, but then changed that and forgot to revert this change.